src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
+## How it works
 
-If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+A service start point should:
+- give the user just enough information to help them understand what the service does and whether it will meet their need - including [giving the service an name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
+- include a ‘button’ linking into the service, with text that’s consistent with the action you’re asking users to take - for example, ‘Start now’, ‘Sign in’ or ‘Register or update your details’ (if you need a secondary call to action, use a standard link)
+- let users sign in, resume an application they’ve already started or update their details (if relevant)
+- include any other information that most users are likely to need before they start using the service online - for example, how much it costs to use the service and roughly how long it will take
+- include details of other ways to access the service — for example, by telephone or by completing a paper form
 
-You can use the coded example here to test what works best for your users in prototypes.
+You’ll also need to list any documents or information the user will need to help them complete the service (there’s no need to list information that a user is likely to know from memory — for example, their own date of birth).
 
-## How it works
+You can list the documents or information at the start point, if it doesn’t make the page too complicated. Or you can move these details to a separate ‘What you’ll need’ page, served within the service subdomain.
+
+## Eligibility
+
+Avoid using the start point to provide complex eligibility information — instead, ask questions inside the service to establish whether the user is eligible.
+
+Read [guidance on designing how GOV.UK content and transactions work together](https://www.gov.uk/service-manual/design/govuk-content-transactions).
+
+## Choosing a format for a start point on GOV.UK
+
+The options available depend on whether the start point will be published as ‘mainstream’ or ‘Whitehall’ content.
+
+The content team at GDS is responsible for creating mainstream start points (as with other mainstream content). ‘Mainstream’ content is content aimed at the general public — or at meeting business needs that aren’t sector-specific.
+
+### Mainstream start points
+
+For a mainstream start point, the options are:
+- a simple start page
+- a start point within a multipart guide
+
+Use a simple start page if users can start using the service without needing much context.
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page, taken from the coronavirus shielding support service. The page includes a heading, a few simple paragraphs and bullet points to explain who can use the service, and a start button labelled 'Register or update your details'.">
```

Comment:
Think this is 279 chars (apparently GOV UK limit is 240).

Reply:
### 
```suggestion
<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page from the coronavirus shielding support service. The page includes a heading, text to explain who can use the service, and a start button labelled 'Register or update your details'.">
```